<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface_Paie_RH</class>
 <widget class="QMainWindow" name="Interface_Paie_RH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des bulletins de salaire</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Traiter les données</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_bull_verifier">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>179</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechercher et exporter PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_etat">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>130</y>
        <width>28</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>601</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Rubriques</string>
      </property>
      <widget class="QLabel" name="label_salaire_fixe">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Salaire fixe  €</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_fixe">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_CSG_deductible">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>107</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>(-) CSG déductible €</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CSG_non_deductible">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>149</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>(-) CSG non déductible €</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_csg">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_1bull_csgnon">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>110</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_hs">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>144</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Heure supplémentaire  €</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_hs">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_bull_id">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_bulletin_id">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>40</y>
        <width>67</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bulletin Id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_bull_periode">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_bull_etat">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>130</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Payee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non payee</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_periode_payee">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>100</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Période payée</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_travailleur_id">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>70</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Travailleur Id</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_prime">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <double>100150.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_prime">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>52</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Prime €</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_co">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <double>100150.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_cotisation">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(-) Cotisations sociales  €</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_crds">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>280</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <double>100150.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_CRDS">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>66</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(-) CRDS €</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_absence">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>200</y>
        <width>143</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(-) Absence non justifiée € </string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_bull_abs">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>200</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <double>100150.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bull_ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bull_modifier">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bull_supp">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QFrame" name="logo_edf">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>169</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/EdfErp.png);
image: url(:/images/logo_edf_erp.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_id_emp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Afficher les données</string>
     </attribute>
     <widget class="QTableView" name="tableView_paie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_paie_actualiser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_bull_id</tabstop>
  <tabstop>lineEdit_bull_periode</tabstop>
  <tabstop>comboBox_bull_etat</tabstop>
  <tabstop>doubleSpinBox_bull_fixe</tabstop>
  <tabstop>doubleSpinBox_bull_prime</tabstop>
  <tabstop>doubleSpinBox_bull_abs</tabstop>
  <tabstop>doubleSpinBox_bull_hs</tabstop>
  <tabstop>doubleSpinBox_bull_co</tabstop>
  <tabstop>doubleSpinBox_bull_csg</tabstop>
  <tabstop>doubleSpinBox_1bull_csgnon</tabstop>
  <tabstop>doubleSpinBox_bull_crds</tabstop>
  <tabstop>pushButton_bull_ajouter</tabstop>
  <tabstop>pushButton_bull_modifier</tabstop>
  <tabstop>pushButton_bull_supp</tabstop>
  <tabstop>pushButton_bull_verifier</tabstop>
  <tabstop>pushButton_paie_actualiser</tabstop>
  <tabstop>tableView_paie</tabstop>
 </tabstops>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
